experiments/list_unsupported_report.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/sh
+
+# Include configuration variables.
+. "$(dirname $0)/build_vars.cfg"
+
+# The script reads two input files, one for standard output and one
+# for standard input. Then concatenates them into the same file. This
+# is done to avoid interleaved text between the two streams breaking
+# the parsing done.
+STDOUT=$1
+STDERR=$2
+file_name="$(cat $STDOUT $STDERR)"
```

Comment:
This looks very bogus.... :-) This is reading the _content_ of the stdout and stderr files into a single shell variable, called `file_name` - however in the next couple of lines you then _use_ that shell variable as if it contains the name of a file.....